lang/system/pl.xml

```diff
@@ -164,22 +226,22 @@ Kod błędu: {}</an_error_occurred>
 			<days>30</days>
 			<days>31</days>
 		</small_mday>
-		<hidden_trophy>Skjult trophy</hidden_trophy>
-		<one_hour_ago>1 godz. temu</one_hour_ago>
-		<one_minute_ago>1 min temu</one_minute_ago>
-		<bronze>Bronse</bronze>
-		<gold>Gull</gold>
-		<platinum>Platina</platinum>
-		<silver>Sølv</silver>
+		<hidden_trophy>Ukryte trofeum</hidden_trophy>
+		<one_hour_ago>godzinę temu</one_hour_ago>
+		<one_minute_ago>minutę temu</one_minute_ago>
+		<bronze>Brązowe</bronze>
+		<gold>Złote</gold>
+		<platinum>Platynowe</platinum>
+		<silver>Srebrne</silver>
 		<hours_ago>{} godz. temu</hours_ago>
-		<minutes_ago>{} min temu</minutes_ago>
+		<minutes_ago>{} min. temu</minutes_ago>
```

Comment:
for it, revert, is already fine, no have . for min word

Reply:
Corrected.